operators/pkg/controller/elasticsearch/validation/validation.go

```diff
@@ -81,7 +82,7 @@ func Validate(es estype.Elasticsearch) ([]validation.Result, error) {
 			Version:       *v,
 		},
 	}
-	var errs []validation.Result
+	var errs []validation.Result //nolint:go-lint
```

Comment:
Is nolint needed?

Reply:
Yeah it complained about not preallocating, though maybe there's a difference in configuration in my editor linter and the one we use for testing. Might need to reconcile that on my side. I'll remove it here if needed